wca-regulations.md

```diff
@@ -467,7 +467,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 
 - D1) Standard speed solving procedures are followed, as described in [Article A](regulations:article:A) (Speed Solving). Additional regulations that supersede the corresponding procedures in [Article A](regulations:article:A) are described below.
     - D1a) During the attempt, the competitor must sit in a chair, sit on the surface, or stand.
-    - D1b) During the attempt, the competitor must use only their feet and the surface to manipulate the puzzle. Penalty: disqualification of the attempt (DNF).
+    - D1b) After starting the timer, the competitor must use only their feet and the surface to manipulate the puzzle. Penalty: disqualification of the attempt (DNF).
```

Comment:
For consistency with [C1b)](https://www.worldcubeassociation.org/regulations/#C1b) I would change

> After starting the timer,

for

> During the solve,

Reply:
Good catch. I can't remember why I changed this, but your suggestion is definitely better.